sopt/src/main/scala/dagr/sopt/cmdline/ClpReflectiveBuilder.scala

```diff
@@ -265,7 +267,11 @@ private[sopt] class ClpArgument(declaringClass: Class[_],
     */
   def toCommandLineString: String = this.value match {
```

Comment:
I see this handles the collections.  I'm not sure why it's broken into two methods though.